eclipsesource/theia
# vscode: support  editor/title/run predefined entry

#### What it does

_Adds a missing predefined entry in the menus_
This enables the possibility to contribute menus to editor/title/run entry as defined in vscode.  

Fixes #9387

Contributed on behalf of ST Microelectronics

#### How to test 
1. Install following extension:
-   extension: [sample-menu-extension-0.0.1.zip](https://github.com/eclipsesource/theia/files/11725066/sample-menu-extension-0.0.1.zip)
- source: [sample-menu-extension-src.zip](https://github.com/eclipsesource/theia/files/11725080/sample-menu-extension-src.zip)
2. Try to open a menu on any file except md file, one menu should be contributed by the extension. A second one will pop-up on any md file. 
![before-addeMenu](https://github.com/eclipsesource/theia/assets/3964263/42893ebc-2a0a-42fc-b3ab-3909faafed40)

3. Update your `theia` repository to use this PR
4. Any file should now also have an additional run menu entry, and the md files shall display a total of 4 menus with this extension.
![after-addeMenu](https://github.com/eclipsesource/theia/assets/3964263/6e6e17f4-5dbf-462e-9ad8-4bc482384ccb)

#### Review checklist


#### Reminder for reviewers
